crates/btc-relay/src/lib.rs

```diff
@@ -753,27 +943,27 @@ impl<T: Config> Module<T> {
 
     /// Get a blockchain from the id
     fn get_block_chain_from_id(chain_id: u32) -> Result<BlockChain, DispatchError> {
-        <ChainsIndex>::get(chain_id).ok_or(Error::<T>::InvalidChainID.into())
+        ChainsIndex::<T>::get(chain_id).ok_or(Error::<T>::InvalidChainID.into())
     }
 
     /// Get the current best block hash
     pub fn get_best_block() -> H256Le {
-        <BestBlock>::get()
+        BestBlock::<T>::get()
     }
 
     /// Check if a best block hash is set
     fn best_block_exists() -> bool {
-        <BestBlock>::exists()
+        BestBlock::<T>::exists()
     }
 
     /// get the best block height
     pub fn get_best_block_height() -> u32 {
```

Comment:
Not necessarily related to this PR but we should convert the chain height to an associated type (i.e. `T::BlockHeight`).

Reply:
Yea I don't think that should be part of this PR - I want this PR to have no functional changes. Please make a ticket if you feel this is important